drizzlepac/runastrodriz.py

```diff
@@ -72,6 +72,8 @@
 
 import math
 
+import math
+
```

Comment:
Why is this import here twice?

Reply:
Actually, it was just one of a few unused import statements which I removed.